renderer/reducers/peripherals.ts

```diff
@@ -32,12 +30,10 @@ export const peripherals = (state: PeripheralState = initialPeripheralState, act
       (action.peripherals ?? []).forEach((peripheral: Peripheral) => {
         if (peripheral.name === consts.PeripheralTypes.BatteryBuzzer) {
           const batteryParams = peripheral.params;
+          //since batterysafety is not included in runtimestatus, have to extract the value from params
           if (batteryParams[IS_UNSAFE] && batteryParams[IS_UNSAFE].bval) {
             nextState.batterySafety = batteryParams[IS_UNSAFE].bval!;
           }
-          if (batteryParams[V_BATT] && batteryParams[V_BATT].fval) {
-            nextState.batteryLevel = batteryParams[V_BATT].fval!;
```

Comment:
Got removed because the batteryLevel is already changed from calling changeBatteryLevel in runtime.ts